focus-ios/focus-ios-tests/XCUITest/TrackingProtectionTest.swift

```diff
@@ -0,0 +1,120 @@
+
+import XCTest
+
+class TrackingProtectionTest: BaseTestCase {
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2544056
+    func testInactiveSettings() {
+        // Go to in-app settings
+        // Check the new options in TP Settings menu
+        dismissURLBarFocused()
+        waitForExistence(app.buttons["HomeView.settingsButton"], timeout: 10)
+        // Set search engine to Google
+        app.buttons["HomeView.settingsButton"].tap()
+        let settingsButton = app.settingsButton
+        waitForExistence(settingsButton, timeout: 10)
+        settingsButton.tap()
+
+        waitForExistence(app.tables.cells["settingsViewController.trackingCell"], timeout: 10)
+        app.tables.cells["settingsViewController.trackingCell"].tap()
+
+        waitForExistence(app.navigationBars["Tracking Protection"])
+        // Verify trackers and scripts to block switches
+        let switchAdvertisingValue = app.switches["BlockerToggle.BlockAds"].value!
+        let switchAnalyticsValue = app.switches["BlockerToggle.BlockAnalytics"].value!
+        let switchSocialValue = app.switches["BlockerToggle.BlockSocial"].value!
+        let switchOtherValue = app.switches["BlockerToggle.BlockOther"].value!
+
+        XCTAssertEqual(switchAdvertisingValue as! String, "1")
+        XCTAssertEqual(switchAnalyticsValue as! String, "1")
+        XCTAssertEqual(switchSocialValue as! String, "1")
+        XCTAssertEqual(switchOtherValue as! String, "0")
+    }
+
+    // Smoketest
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/394999
+    func testProtectionSidebar() {
+        // Visit https://www.mozilla.org
+        loadWebPage("mozilla.org")
+
+        // Check the correct site is reached
+        waitForWebPageLoad()
+
+        // Open the tracking protection sidebar
+        app.buttons["URLBar.trackingProtectionIcon"].tap()
+
+        // Disable tracking protection
+        waitForExistence(app.switches["BlockerToggle.TrackingProtection"])
+        app.switches["BlockerToggle.TrackingProtection"].tap()
+
+        // Reopen the tracking protection sidebar
+        if !iPad() {
+            app.buttons["closeSheetButton"].tap()
+            app.buttons["URLBar.trackingProtectionIcon"].tap()
+        }
+
+        // Wait for the sidebar to open
+        let switchValue = app.switches["BlockerToggle.TrackingProtection"].value!
+        XCTAssertLessThan(switchValue as! String, "2")
+    }
+
+    // Smoke test
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/1569890
+    func testAdBlocking() {
+        // Load URL
+        loadWebPage("https://blockads.fivefilters.org/")
+        waitForWebPageLoad()
+
+        // Check ad blocking is enabled
+        let TrackingProtection = app.staticTexts["Ad blocking enabled!"]
+        XCTAssertTrue(TrackingProtection.exists)
+    }
+
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/1569869
+    func testShieldMenuSetting() {
```

Comment:
I have updated the "Automation" Status of the TestRail test.
https://testrail.stage.mozaws.net/index.php?/cases/view/1569869